src/profile/Index.vue

```diff
@@ -1,89 +1,171 @@
+                    <v-list-item-subtitle v-html="item.secondary"></v-list-item-subtitle>
+                  </v-list-item-content>
 
-      <v-col cols="12" sm="6" md="4">
-        <DoNotDiscloseCard :dnd="$user.hide"/>
+                  <v-list-item-avatar>
+                    <v-icon>mdi-launch</v-icon>
+                  </v-list-item-avatar>
+                </v-list-item>
+              </a>
+            </template>
+          </v-card>
+        </v-list>
       </v-col>
     </v-row>
 
-    <v-subheader>{{ $vuetify.lang.t('$vuetify.profile.index.2sv') }}</v-subheader>
+    <v-row>
+      <v-col>
+        <h3>Google Workspace</h3>
+      </v-col>
+    </v-row>
 
     <v-row>
-      <v-col cols="12" sm="6" md="4">
-        <TotpCard :meta="mfa.totp"/>
+      <v-col>
+        <h5>Login with your {{org}}'s Google account</h5>
       </v-col>
+    </v-row>
 
-      <v-col cols="12" sm="6" md="4">
-        <SecurityKeyCard :meta="mfa.webauthn"/>
+    <v-row class="my-10">
+      <v-col>
+        <div class="flex row justify-space-around ml-2">
+          <a v-for="(item, index) in googleItems" :key="index" :href="item.url" target="_blank" class="text-decoration-none">
+            <v-img :id="item.label" :src="item.image" max-width="40px" class="mx-auto"/>
+            <label :for="item.label">{{item.label}}</label>
+          </a>
+        </div>
       </v-col>
+    </v-row>
 
-      <v-col cols="12" sm="6" md="4">
-        <BackupCodeCard :meta="mfa.backup"/>
+    <v-row>
+      <v-col>
+        <v-footer>
+        <small>
+          <span v-for="(item, index) in googleItems" :key="index">
+            {{item.label + getSpaceCommaOrAnd(index, googleItems.length)}}
+          </span> are trademarks of Google LLC.
+        </small>
+        </v-footer>
       </v-col>
     </v-row>
   </BasePage>
 </template>
 
 <script>
 import ProfileProgress from './ProfileProgress'
-import PasswordCard from './PasswordCard'
-import PasswordRecoveryCard from './PasswordRecoveryCard'
-import TotpCard from './TotpCard'
-import SecurityKeyCard from './SecurityKeyCard'
-import BackupCodeCard from './BackupCodeCard'
-import DoNotDiscloseCard from './DoNotDiscloseCard'
-import Attribute from './Attribute'
-import { recoveryMethods, retrieve as retrieveMethods} from '@/global/recoveryMethods';
-import { mfa, retrieve as retrieveMfa } from '@/global/mfa';
+import { recoveryMethods, retrieve as retrieveMethods} from '@/global/recoveryMethods'
+import { mfa, retrieve as retrieveMfa } from '@/global/mfa'
+
 
 export default {
   components: {
-    ProfileProgress,
-    PasswordCard,
-    PasswordRecoveryCard,
-    TotpCard,
-    SecurityKeyCard,
-    BackupCodeCard,
-    DoNotDiscloseCard,
-    Attribute,
+    ProfileProgress
   },
   data: () => ({
     alternates: recoveryMethods.alternates,
     mfa,
   }),
   computed: {
-    hasUnverifiedEmails: vm => vm.alternates.some(m => ! m.verified),
+    org: (vm) => vm.$user.email.split('@')[1].split('.')[0],
```

Comment:
We may want to define the organization display name in the skinny repo's `.env` as well, rather than trying to extract it from the user's email address.

In part, this is because the capitalization isn't identical across organization names, and in part because sometimes an organization's IDP can have a user with an email address provided by a different organization (depending on what the HR system provides as the primary email address for a user). For example, some of our IDPs used for providing access to partners allow things like `@gmail.com` email addresses.